modules/service-accounts-as-oauth-clients.adoc

```diff
@@ -110,9 +111,9 @@ reference object. For example:
 ----
```

Comment:
For lines 70-81 and 87-99, should they be updated to syntax highlight as JSON (e.g., `[source,json]`)?

Reply:
I considered it, but wasn't 100% on whether to add them to the code blocks above and below, so left it alone for now.